submissions/bilkisuTace/database/init.sql

```diff
@@ -0,0 +1,96 @@
+-- ----------------------------
+-- Table structure for addresses
+-- ----------------------------
+CREATE TABLE addresses (
+  customer_id int(11) NOT NULL,
+  street_address varchar(255) NOT NULL,
+  city varchar(255) NOT NULL,
+  state varchar(255) NOT NULL,
+  zipcode int(11) DEFAULT NULL,
+  country varchar(255) NOT NULL,
+  mobile_phone varchar(255) NOT NULL,
+  PRIMARY KEY (customer_id),
+  CONSTRAINT addresses_fk_1 FOREIGN KEY (customer_id) REFERENCES customers (customer_id) ON DELETE CASCADE ON UPDATE CASCADE
```

Comment:
You need to create a `customers` table before the `addresses` table, so you can easily reference it as a foreign key here